README.md

```diff
@@ -4,31 +4,123 @@
 [![Build Status][circle-image]][circle-url]
 [![Known Vulnerabilities][snyk-image]][snyk-url]
 
-**WIP** [pprof][pprof-url] support for Node.js.
+[pprof][pprof-url] support for Node.js.
 
-# Running the system test
-The system test starts a simple benchmark, uses this module to collect a time
-and a heap profile, and then verifies that the profiles can be opened and that
-the profiles contain functions from within the benchmark. 
+## Prerequisites
+1. Your application will need to be using Node.js 8.9.4 or greater, or Node.js 
+10.4.1 or greater. The profiler will not be enabled when using earlier versions
+of Node 8 and 10 because the profiler is not stable with those versions of
+Node.js.
+    * Versions of Node.js before Node.js 8 prior to 8.9.4 are impacted by
```

Comment:
Versions of Node.js prior to 8.9.4 are affected

Reply:
Done.